examples/viewer.py

```diff
@@ -73,6 +76,50 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
             key.Z: "move_up",
         }
 
+        # Load a TrueTypeFont plugin and open the font
+        self._font = text.FontManager().load_and_instantiate("TrueTypeFont")
+        self._font.open_file(
+            os.path.join(os.path.dirname(__file__), "Cousine-Regular.ttf"), 180.0
+        )
+
+        # Glyphs we need to render everything
+        self._cache = text.DistanceFieldGlyphCache(
+            mn.Vector2i(2048), mn.Vector2i(512), 22
+        )
+        self._font.fill_glyph_cache(
+            self._cache,
+            "abcdefghijklmnopqrstuvwxyz"
+            "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
```suggestion
            string.ascii_lowercase,
            string.ascii_uppercase,
```
You will need to `import string`